tests/testthat/model-examples/1/1.cpu

```diff
@@ -1 +1 @@
-       2.543
+       2.696
```

Comment:
Do we still need this file? I don't see any references to it.

Reply:
Yes, bbi summary parses it. If it's missing it will throw an error.